sfcasts/non-autowirable-services.md

```diff
@@ -1,3 +1,32 @@
\ No newline at end of file
+In the last chapter, we autowired a *non-autowireable* argument. *This time*, let's try to autowire an non-autowireable *service*. But before we do that, we need to *find* a non-autowireable service. To do that, at your terminal, run:
+
+```terminal
+bin/console debug:container
+```
+
+If you think this Twig service can't be autowired since it's just an ID, *think again*. If we scroll up, we see `Twig\Environment`. *This* is an alias for our Twig service. *Conversely*, `twig.command.debug` is *not* autowireable. This is the service that powers the
+
+```terminal
+debug:twig
+```
+
+command we used in previous chapters. When we run that in our terminal,
```

Comment:
```suggestion
If you think this Twig service can't be autowired since it's just an ID, *think again*. If we scroll up, we see `Twig\Environment`. *This* is an alias for our Twig service. *Conversely*, `twig.command.debug` is *not* autowireable. This is the service that powers the `debug:twig` command we used in previous chapters. When we run that in our terminal,
```